client/me/purchases/purchase-item/index.jsx

```diff
@@ -66,6 +66,27 @@ class PurchaseItem extends Component {
 	getStatus() {
 		const { purchase, translate, locale, moment, name, isJetpack, isDisconnectedSite } = this.props;
 		const expiry = moment( purchase.expiryDate );
+		// To-do: There isn't currently a way to get the taxName based on the country.
+		// The country is not includedin the purchase information envelope
+		// We should add this information so we can utilize useTaxName to retrieve the correct taxName
```

Comment:
Interesting! Receipts (eg: from the endpoint `/me/billing-history/past` or the single receipt endpoint) do include this information with the `tax_country_code` property, but "purchases" (which I guess means subscriptions, from the endpoint `/sites/188830194/purchases`) do not. Since each subscription should have an associated receipt, we should be able to modify the endpoint to include the tax data. But yes, it could be done as a follow-up to this change.

Reply:
👍  I'll make an issue for it.